taskweaver/module/prompt_util.py

```diff
@@ -0,0 +1,54 @@
+from typing import List, Tuple
+
+
+class PromptUtil:
+    DELIMITER_TEMPORAL: Tuple[str, str] = ("{{DELIMITER_START_TEMPORAL}}", "{{DELIMITER_END_TEMPORAL}}")
```

Comment:
wondering whether we should make this configurable in case in some scenario, the documents or data that would conflict with these pairs

Reply:
let's try it as it is now and extend it in the future when there is a real external requirement